pangeo/values.yaml

```diff
@@ -7,6 +7,9 @@
 rbac:
   enabled: true
 
+# Create Prometheus and Grafana Services
+monitoring:
+  enabled: true
```

Comment:
The requirements.yaml file can use this as an indication to install the requirement or not, so that a disabled monitoring doesn't lead to installation of prometheus/grafana.

See: https://helm.sh/docs/topics/charts/#tags-and-condition-fields-in-dependencies